packages/client/src/components/app/SidePanel.svelte

```diff
@@ -26,6 +29,10 @@
     }
   }
 
+  $: {
+    sidePanelStore.actions.setSidepanelState(clickOutsideToClose)
```

Comment:
We can't use this store to store a single boolean here, because each side panel has its own setting for whether it can be clicked outside to close or not. By using a single store, multiple side panels will override each other and the last to render will "win" in case where we have multiple side panels (which we do have - like the table autoscreen).

One solution would be only setting this value when a certain side panel actually opens - because all we care about is handling the correct behaviour for the side panel that is currently open, and only one can be open at any given time.

The way you could do this is move this line to be inside the
```js
if (open) {
...
}
```
statement down below.

This way the store is updated to reflect the desired behaviour whenever any side panel is opened.